lib/game_statistics.rb

```diff
@@ -1,24 +1,23 @@
-require_relative "game_data"
-class GameStatistics
+require_relative "futbol_data"
+
+class GameStatistics < FutbolData
```

Comment:
Inheritance. Awesome!

Reply:
YAY! Not only can we check this off our rubric, but I feel so much better about these and being able to track the code!